### ISSUE TITLE
Style Request - More whitespace for json blobs with nested dicts

### ISSUE THREAD

**Issue by phumke** [grantjenks/blue#55]
I have a black formatted json blob that looks like the following:
```
    return jsonify(data={
        'id': 123,
        'name': 'A Content Type',
        'type': 'json',
        'field': [
            {
                'name': 'headline',
                'type': 'string'
            },
            {
                'name': 'url',
                'type': 'string'
            }
        ]
    })
```

After formatting with blue the formatting was changed to the following:
```
    return jsonify(
        data={
            'id': 123,
            'name': 'A Content Type',
            'type': 'json',
            'field': [{'name': 'headline', 'type': 'string'}, {'name': 'url', 'type': 'string'}],
        }
    )
```

My own personal preference is towards the original formatting since it's much easier to identify field count and pick out the name fields at a glance.

**Comment by warsaw** [comment]
I personally think getting formatting like this right 100% of the time is impossible.  I suggest [adding explicit skip comments](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html?highlight=fmt%3A%20off) to prevent reformatting those lines.

**Comment by phumke** [comment]
Yeah that's fair - especially given the number of permutations here.  The explicit skip helps me in this instance, thank you.

I like the idea of `# fmt: black/blue` but what about including a config option for `json: black/blue - default blue`.  That way all json blobs in the codebase are formatted consistently.

**Comment by warsaw** [comment]
I guess the problem with that is that blue (or black AFAIK) doesn't really semantically know that this is JSON.  It just looks like a function call with an argument set to a dictionary.

**Comment by phumke** [comment]
Yeah good point - instead of json, maybe a `dict: blue/black` option?

**Comment by grantjenks** [comment]
Are you on the most recent version of black? I’m guessing there’s something different about the trailing comma handling. Notice that the “type” field doesn’t have a trailing comma in the first post.

**Comment by phumke** [comment]
That's a good eye, the code was originally formatted using black v `20.8b1`
